docs/docker-hub/userguide.md

```diff
@@ -33,7 +33,7 @@ you can look at your profile page on [Docker Hub](https://hub.docker.com).
 
 ## Pulling, running and building images
 
-You can find more information on [working with Docker images](../userguide/dockerimages.md).
+You can find more information on [working with Docker images](/userguide/dockerimages/).
```

Comment:
so, for these, we need to use an absolute path and can't use `.md`? (just a question)